### ISSUE TITLE
Conda

### ISSUE THREAD

**Issue by liliblu** [sknetwork-team/scikit-network#238]
Is there any plan to make this package available on a conda channel?

**Comment by QLutz** [comment]
Hello Lili,

We are not familiar with Conda channels.

Could you explain what you need them for and give us a link explaining how to make sknetwork available on a conda channel?

Thanks in advance.

**Comment by liliblu** [comment]
Hi, thanks for getting back to me. 
I think the upside to putting your package on a conda channel is that it makes it easier to add as a dependency to other conda packages. Many packages will be on pip and conda at the same time. 
Here is a link about contributing to conda-forge, a popular channel:
https://conda-forge.org/docs/maintainer/adding_pkgs.html

**Comment by QLutz** [comment]
Hi,

We are going to give this a go.

Thank you for your interest in our package!

**Comment by parashardhapola** [comment]
Hi,

Wondering if any progress has been made on this?

I'm using `scikit-network` as a dependency for a package that I'm co-developing: [scarf](https://scarf.readthedocs.io/en/latest/)
Among 33 dependencies of `scarf` only 2 are not on [conda-forge](https://conda-forge.org/docs/maintainer/adding_pkgs.html#) and `scikit-network` is one of those two.

I tried to generate a recipe for `scikit-network` [here](www.marcelotrevisani.com/grayskull) but for some reason it failed. It could be a good place to start. 

Please let me know if you'd like me look further into this.

**Comment by QLutz** [comment]
Hi,

We've made changes as asked in PR #486 (albeit adapted for our recent transition to GitHub Actions).

Please do tell us if that is any help.

**Comment by parashardhapola** [comment]
Not sure what you meant there.
In PR #486 a github workflow was added to build wheels. But these wheels are built locally and not really available after the 'action' has finished. However, these builds are used when a PyPi release is made. 

I have created pull request on conda-forge [here](https://github.com/conda-forge/staged-recipes/pull/15753)
Let's see if this builds successfully on conda-forges CI.
